packages/extension-ui/src/Popup/About.tsx

```diff
@@ -62,18 +59,13 @@ const About = () => {
               title={t<string>('Visit Website')}
             />
           </LinksList.Group>
-        </Nav>
+        </nav>
       </Container>
     </>
   );
 };
 
-export default About;
-
 const Container = styled.div`
-  display: flex;
-  flex-direction: column;
-  justify-content: space-evenly;
```

Comment:
Why remove this? It gives a nice, window-size-agnostic layout.

Reply:
I'm not sure if spacing evenly works better than constant margin for greater height. It's true in some cases, but here the gap might get awkward :shrug: If you think it's important I can change this, but we aren't interested in different screen sizes anyway, right?

Reply:
Not for this particular extension popup view, no; but since we do account for _some_ views (external popups) resizing, it's better to have all screens prepared for such a possibility, if the cost is not high, in case a view was to change a window type (which we already have experienced happening) in which case it'd be very easy to forget to adapt the styles. That said I don't have a strong opinion about whether margins would look better than spacing evenly in that particular case, so I'll leave the decision up to you :)